patterns/services-three-column-boxed-pricing-table.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Title: Services - Three-column boxed pricing table
+ * Slug: twentytwentyfive/services-three-column-boxed-pricing-table
+ * Categories: call-to-action, banner, featured, services
+ * Description: A three-column boxed pricing table designed to showcase services, descriptions, and pricing options.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"margin":{"top":"0","bottom":"0"},"blockGap":"0px"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)"><!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide"><!-- wp:heading {"fontSize":"x-large"} -->
+<h2 class="wp-block-heading has-x-large-font-size">Choose your membership</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"className":"is-style-pill"} -->
+<p class="is-style-pill">Become a member</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group -->
+
+<!-- wp:columns {"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|70","bottom":"0"}}}} -->
+<div class="wp-block-columns alignwide" style="margin-top:var(--wp--preset--spacing--70);margin-bottom:0"><!-- wp:column {"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-column"><!-- wp:group {"style":{"border":{"width":"1px","color":"#11111133","radius":"10px"},"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40","left":"var:preset|spacing|40","right":"var:preset|spacing|40"}}},"layout":{"type":"default"}} -->
```

Comment:
Can you please move the border styles to the columns and use the preset of opacity-20 for the color? the same for the other columns.